code/game/objects/items/devices/scanners.dm

```diff
@@ -23,26 +23,43 @@ BREATH ANALYZER
 	var/mode = 1
 
 /obj/item/device/healthanalyzer/attack(mob/living/M as mob, mob/living/user as mob)
-	health_scan_mob(M, user)
+	health_scan_mob(M, user, FALSE)
 	src.add_fingerprint(user)
 	return
 
 /obj/item/device/healthanalyzer/attack_self(mob/user)
-	health_scan_mob(user, user)
+	health_scan_mob(user, user, FALSE)
 	src.add_fingerprint(user)
 	return
 
-/proc/health_scan_mob(var/mob/living/M, var/mob/living/user, var/visible_msg, var/ignore_clumsiness, var/show_limb_damage = TRUE)
-	if ( ((user.is_clumsy()) || (DUMB in user.mutations)) && prob(50))
+/proc/calcDamage(var/DMGValue)
+	switch(DMGValue)
+		if (0 to 1)
+			return "Healthy"
+		if (1 to 10)
+			return "Negligible"
+		if (10 to 25)
+			return "Minor"
+		if (25 to 50)
+			return "Moderate"
+		if (50 to 75)
+			return "Major"
+		if (75 to 200)
+			return "Critical"
+		if (200 to INFINITY)
+			return "Fatal"
+
+/proc/health_scan_mob(var/mob/living/M, var/mob/living/user, var/adv, var/visible_msg, var/ignore_clumsiness, var/show_limb_damage = TRUE)
+	if (((user.is_clumsy()) || (DUMB in user.mutations)) && prob(50))
 		to_chat(user, text("<span class='warning'>You try to analyze the floor's vitals!</span>"))
 		for(var/mob/O in viewers(M, null))
 			O.show_message("<span class='warning'>\The [user] has analyzed the floor's vitals!</span>", 1)
-		user.show_message("<span class='notice'>Analyzing Results for The floor:</span>", 1)
-		user.show_message("Overall Status: Healthy</span>", 1)
-		user.show_message("<span class='notice'>    Damage Specifics: 0-0-0-0</span>", 1)
-		user.show_message("<span class='notice'>Key: Suffocation/Toxin/Burns/Brute</span>", 1)
-		user.show_message("<span class='notice'>Body Temperature: ???</span>", 1)
-		return
+			user.show_message("<span class='notice'>Analyzing Results for The floor:</span>", 1)

```

Comment:
If you add that indentation there, then the user will get the Analyzing results for the floor one time for every mob that gets returned by viewers.
Not sure if thats the goal here.

Reply:
Good catch, fixing now.